src/StakingValidator.hs

```diff
@@ -51,6 +54,7 @@ data RouterRedeemer = RouterRedeemer
 PlutusTx.makeLift ''RouterRedeemer
 PlutusTx.makeIsDataIndexed ''RouterRedeemer [('RouterRedeemer, 0)]
 
+-- TODO: Switch to: '[ "indices" ':= PBuiltinList (PAsData (PBuiltinPair (PAsData PInteger) (PAsData PInteger)))
```

Comment:
Is this a missed thing, or for the future?

Reply:
It's for future, to unify the interface with our Aiken's. I believe having a single list of pairs signals the underlying logic better. Although I'm not sure if it'd impact the performance negatively.

If having two lists is cheaper, perhaps we should refine the Aiken's interface?